### PR TITLE
Added fedora-review tool for rdo packages blog

### PR DESCRIPTION
**Pull request by chkumar246**

### CODE REVIEW

**Comment by chkumar246** [comment]
@apevec @javierpena @hguemar Please have a look.

File: source/blog/2016-08-03-fedora-review-tool-for-reviewing-rdo-packages.html.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+	$ sudo yum -y install https://kojipkgs.fedoraproject.org//packages/fedora-review/0.5.3/2.el7/noarch/fedora-review-0.5.3-2.el7.noarch.rpm
+
+**On Fedora 24**
+
+	$ sudo dnf -y install fedora-review
+
+[2.] Add the user you intend to run as to the mock group:
+
+	$ sudo usermod -a -G mock $USER
```
Comment (javierpena) [inline source/blog/2016-08-03-fedora-review-tool-for-reviewing-rdo-packages.html.md]: mock needs to be installed, so we can add the user to the group. Maybe we should install it at the sme time as fedora-review?

**Comment by javierpena** [comment]
Other than the inline comment, it's :+1: for me.

**Comment by chkumar246** [comment]
@javierpena Fixed, Thanks :-)
@apevec @hguemar Any more suggestion to improve this blog.

**Comment by javierpena** [comment]
:+1:

**Pull request merged**